src/test/java/uk/gov/companieshouse/reconciliation/email/SendElasticsearchEmailRouteTest.java

```diff
@@ -76,12 +76,17 @@ public void configure() throws Exception {
                 .withHeader("ResourceLinkReference", "Compare Status Primary Mongo Link")
                 .build();
 
+        Exchange sixthExchange = ExchangeBuilder.anExchange(context)
```

Comment:
Please use a more meaningful variable name here.

Reply:
Resolved by Kin

Reply:
Resolved in latest commit.